### ISSUE TITLE
FR: Menu option to quickly hook into/unhook from `bookshelf.app` on Pocketbook

### ISSUE THREAD

**Issue by sdasda7777** [koreader/koreader#11742]
**Does your feature request involve difficulty completing a task? Please describe.**
Hi, I noticed some time ago that when you copy `system/bin/koreader.app` into `system/bin/bookshelf.app`, Koreader starts behaving as the main application, i.e. it starts on startup and home button doesn't exit it anymore. I think this is really neat, but the downside is that USB connection to PC doesn't work when this is applied, which is why I have to rename the file when transfering books, and then rename it back.

**Describe the solution you'd like**
Would it be possible to add a Pocketbook specific button that would create this file if it does not exist and destroy it if it does? I think this probably shouldn't be too difficult and I'd be happy to make a pull request, if you point me to how to make the code specific to pocketbooks.

**Describe alternatives you've considered**
I guess it might be possible to make Koreader work with the USB, but I assume that would be significantly more work.

**Comment by sdasda7777** [comment]
Actually, I'm personally just interested in Koreader being launched at startup. Maybe there is a smarter way to do that, a way that wouldn't disable the home button, but I am not aware of it at the moment.

**Comment by sdasda7777** [comment]
I have a suspicion it might be possible to manipulate the `/mnt/ext1/system/state/current` to do this without loss of functionality, however when I searched for this file online, I came up with #6597 which seems to suggest Koreader already utilizes this file in some way. I can't seem to track down anything closer, though.

**Comment by Frenzie** [comment]
In the PocketBook settings somewhere you can choose home screen (or whatever it's called) or last book on startup. Sounds like the latter will automatically open KOReader.

**Comment by sdasda7777** [comment]
Yeah, that would be absolutely amazing. I don't observe this, though; with the "last book on startup" option selected it opens it in the Pocketbook reader app. Maybe it has to be enabled in the Koreader? But I didn't find anything, so I'm not sure.

**Comment by Frenzie** [comment]
I don't really know PB, but if you mean that KOReader opens a particular type of file by default normally while it doesn't on startup something must've changed. Pinging @rjd22

**Comment by sdasda7777** [comment]
Oh, does that mean you have to set up the default app to open files with for it to work? That sort of would make sense, but I haven't really seen that option anywhere either.

**Comment by Frenzie** [comment]
See https://github.com/koreader/koreader/wiki/Installation-on-PocketBook-devices#integration-with-pocketbook-os-optional

**Comment by sdasda7777** [comment]
That's awesome, thanks!
